app/assets/stylesheets/sephora_style_guide/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,42 @@
+.breadcrumb {
+  list-style: none;
+  display: flex;
+  font-size: 11px;
+  height: 11px;
+  padding: 0;
+}
+
+.breadcrumb-item {
+  display: inline-table;
```

Comment:
When viewed on mobile (iphone 5 specifically), the flex display inherited by the children throws the alignment off. So, I gave them their own display to fix the alignment.